### ISSUE TITLE
[connectivity] TODOs

### ISSUE THREAD

**Issue by cindychip** [lowRISC/opentitan#5907]
- [x]  README (PR: #5913)

- [x]  VC Formal error msg (PR: https://github.com/lowRISC/opentitan/pull/5967)

- [x] Set up connectivity to nightly dashboard (PR:https://github.com/lowRISC/opentitan/pull/6087)

- [ ] Connectivity test toggle coverage (right not it does not coverage)

- [x] batch script, if not go to hw/formal, it gets a wrong path.

- [ ] connectivity and simulation coverage merge.  

- [x] GUI mode also provides a result summary.

**Comment by cindychip** [comment]
@sriyerg @matutem for the visibility. Please feel free to add more feedback and support requests :)

**Comment by msfschaffner** [comment]
Can this issue be updated @cindychip?

**Comment by cindychip** [comment]
Thanks Michael, updated and the remaining two are still WIP ;)

**Comment by msfschaffner** [comment]
Is this needed for M2 or can we defer into M3?

**Comment by msfschaffner** [comment]
@sriyerg is this something we need for M2 or can we do this as part of coverage closure in V3/M3?

**Comment by cindychip** [comment]
Hey Michael, thanks for the reminder. I think this can be post-M2. But feel free to chime in Sri :)

**Comment by sriyerg** [comment]
```
- Connectivity test toggle coverage (right not it does not coverage)
```
We have already addressed some of this (there are exclusions in place at the chip level for things that we verified with connectivity). I believe this is in-progress activity and I (we) will keep adding exclusions to address this. The chip level coverage is already looking pretty good, so at the time of M2 sign-off, we can take a repeat call. 

```
- connectivity and simulation coverage merge.
```
I would consider this to be out of scope for our project. Lets not worry about this. 

In conclusion, we can close this bug. Any objections?

**Comment by cindychip** [comment]
Sure, we can close it.
